SWLOR.Game.Server/Feature/SpawnDefinition/DantooineSpawnDefinition.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Service.SpawnService;
+using Random = SWLOR.Game.Server.Service.Random;
+
+namespace SWLOR.Game.Server.Feature.SpawnDefinition
+{
+    public class DantooineSpawnDefinition : ISpawnListDefinition
+    {
+        private readonly SpawnTableBuilder _builder = new();
+
+        public Dictionary<string, SpawnTable> BuildSpawnTables()
+        {
+            DantooineKinrathCaves();
+            DantooineJungle();
+            DantooineRuinFarm();
+            DanLake();
+            Janta();
+            DanBoss();
+            DanBoss2();
+            DantooineMountains();
+            DantooineCanyon();
+            DantooineSouthPlains();
+            DantooineNpcs();
+
+
+            return _builder.Build();
+        }
+
+        private void DantooineKinrathCaves()
+        {
+            _builder.Create("DANTOOINE_KINRATH_CAVES")
+                .AddSpawn(ObjectType.Creature, "hkinrath")
+                .RandomlyWalks()
+                .WithFrequency(100)
+            .ReturnsHome();
+        }
+
+        private void DantooineJungle()
+        {
+            _builder.Create("DANTOOINE_JUNGLE")
+                .AddSpawn(ObjectType.Creature, "gizka")
+                .RandomlyWalks()
+                .WithFrequency(100)
+
+                .AddSpawn(ObjectType.Creature, "hkinrath")
+                .RandomlyWalks()
+                .WithFrequency(20)
+                .ReturnsHome()
```

Comment:
When doing multiple entries for the same spawn table, add an empty line to make it clear it's the start of a new spawn.